src/solver/hydro/management/management.cpp

```diff
@@ -328,7 +320,7 @@ bool HydroManagement::checkHourlyMinGeneration(uint tsIndex, Data::Area& area) c
                     logs.error()
                         << "In area: " << area.name << " [hourly] minimum generation of "
                         << srcmingen[day * 24 + h] << " MW in timestep " << day * 24 + h + 1
-                        << " of TS-" << tsIndex + 1
+                        << " of TS-" << year + 1
```

Comment:
We either need to change the log, or to retrieve the corresponding TS index here (unfortunately). Better to get the TS index, so the user understands what data it is.
Otherwise it's misleading: when reading the log, we will understand it's the TS number, whereas it's actually the MC year.